### PR TITLE
feat/ adding support for all the possible tick sizes

### PR DESCRIPTION
**Pull request by poly-rodr**
Adding support for:
- `0.1`
- `0.01` - already supported
- `0.001`
- `0.0001`

### Example
```ts
const YES = "1343197538147866997676250008839231694243646439454152539053893078719042421992";
const order = await clobClient.createOrder({
    tokenID: YES,
    price: 0.0005, // <---- price
    side: Side.BUY,
    size: 41,
    feeRateBps: 0,
    nonce: 0,
    expiration: 0,
});
console.log("Created Order", order);

// Send it to the server
const resp = await clobClient.postOrder(order);
console.log(resp);
```

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -347,19 +364,23 @@ export class ClobClient {
     public async createOrder(userOrder: UserOrder): Promise<SignedOrder> {
         this.canL1Auth();
 
-        return this.orderBuilder.buildOrder(userOrder);
+        const tickSize = await this.getTickSize(userOrder.tokenID);
```
Comment (L-Kov) [inline src/client.ts:367]: I think we should allow this to be passed as a param in which case we don't make the api call. The FE will likely already have the tick size available at order creation time.

File: src/client.ts [hunk 2]
```diff
@@ -347,19 +364,23 @@ export class ClobClient {
...
+
+        return this.orderBuilder.buildOrder(userOrder, tickSize);
     }
 
     public async createMarketBuyOrder(userMarketOrder: UserMarketOrder): Promise<SignedOrder> {
         this.canL1Auth();
 
+        const { tokenID } = userMarketOrder;
+        const tickSize = await this.getTickSize(tokenID);
```
Comment (L-Kov) [inline src/client.ts:376]: same here

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@polymarket/clob-client",
     "description": "Typescript client for Polymarket's CLOB",
-    "version": "1.5.1",
+    "version": "1.6.0",
```
Comment (L-Kov) [inline package.json:4]: will likely end up being 2.1.0. There is another PR open (#80) with a published package that will roll out soon and it is breaking so it bumps the major version

File: src/client.ts (hunk 1 above)
Comment (poly-rodr) [reply src/client.ts]: New parameter added ✅

File: src/client.ts (hunk 2 above)
Comment (poly-rodr) [reply src/client.ts]: New parameter added ✅

File: package.json (hunk 3 above)
Comment (poly-rodr) [reply package.json]: done

**Review by L-Kov — APPROVED** [review]

**Pull request merged**